CMakeLists.txt

```diff
@@ -186,6 +186,29 @@ endif()
 
 target_link_libraries(QGroundControl PRIVATE qgc)
 
+option(ENABLE_INSTALL_TARGET "Enable real installation of QGC when running install" OFF)
```

Comment:
Why is this an option ? If someone is using `make install` in QGC it should install QGC.
This should be the default behavior.

Reply:
Since I'm not sure whether other projects or any of our distributions (Windows installer, Linux AppImage, etc.) will hit any issues.
So I added the option as I need it explicitly for my Flatpak builds.
But yes, I can remove it and see what the CI/CD says.
I'm just not able test the macOS dmg..

Or wait.. is qmake not being used everyhere at the moment?

Reply:
While crossing fingers - fixed via: https://github.com/mavlink/qgroundcontrol/pull/9569/commits/2a6af31adfadbe4095ad10e275b892d74dbb344d